### PR TITLE
Side exercises for scrabble part2

### PR DESCRIPTION
**Pull request by glennj**
| exercise | canonical data | difficulty |
| --- | --- | :-: |
| [pig-latin](https://github.com/glennj/tcl/blob/side-exercises-for-scrabble-part2/exercises/pig-latin/README.md) | [canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/pig-latin/canonical-data.json) | 3 |
| [protein-translation](https://github.com/glennj/tcl/blob/side-exercises-for-scrabble-part2/exercises/protein-translation/README.md) | [canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/protein-translation/canonical-data.json) | 3 |
| [secret-handshake](https://github.com/glennj/tcl/blob/side-exercises-for-scrabble-part2/exercises/secret-handshake/README.md) | [canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/secret-handshake/canonical-data.json) | 4 |
| [wordy](https://github.com/glennj/tcl/blob/side-exercises-for-scrabble-part2/exercises/wordy/README.md) | [canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/wordy/canonical-data.json) | 4 |

This is part 2 of 2.

### CODE REVIEW

File: exercises/pig-latin/.meta/version [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1.2.0
+
```
Comment (sshine) [inline exercises/pig-latin/.meta/version]: Unwarranted linebreak.

```suggestion
1.2.0
```

File: exercises/wordy/example.tcl [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    while {[llength $exp] > 1} {
+        # replace "a + b * c" with "[+ a b] * c" ...
+        if {![regsub -expanded $subexpr $exp {[\2 \1 \3]} cmd]} {
+            error "syntax error"
+        }
+        # ... and then perform the command substitution
+        set exp [subst $cmd]
+    }
+    return $exp
+}
```
Comment (sshine) [inline exercises/wordy/example.tcl]: An alternative solution would add parentheses such that the reduced expression can be `expr`'ed.

File: exercises/pig-latin/.meta/version (hunk 1 above)
Comment (glennj) [reply exercises/pig-latin/.meta/version]: I force pushed, can you check again?

**Comment by sshine** [comment]
It's good now.

**Comment by glennj** [comment]
"Rebase and merge" is the way to go, right?

**Comment by sshine** [comment]
Yup!

**Pull request merged**